x-pack/plugins/infra/public/components/asset_details/tabs/overview/alerts.tsx

```diff
@@ -48,13 +53,23 @@ export const AlertsSummaryContent = ({
     [assetName, dateRange]
   );
 
+  const onLoaded = (alertsCount?: AlertsCount) => {
+    const hasActiveAlerts =
+      typeof alertsCount?.activeAlertCount === 'number' && alertsCount?.activeAlertCount > 0;
```

Comment:
We have the type `number | undefined` so the first check is to make sure that it's a number and not `undefined` (Typescript will complain that it can be `undefined` if we remove the check) Do you think there is something wrong with checking that? We can also change it to `alertsCount?.activeAlertCount && ...` but I think checking the type is easy to understand and helps validate the `activeAlertCount` value type I think.

Reply:
suggestion: you could do this

```ts
const { activeAlertCount = 0 } = alertsCount ?? {};
const hasActiveAlerts = activeAlertCount > 0;
...
```

Reply:
Done ✅